test/index.ts

```diff
@@ -47,7 +47,7 @@ function setupCoverage() {
 const mocha = new Mocha({
     ui: "bdd",
     color: true,
-    timeout: 30 * 1000, // for windows extension activation test
+    timeout: 60 * 1000, // for windows extension activation test
```

Comment:
nit: This seems alright but I hope there is a better way to handle these timeouts dynamically.